sharding/collation.go

```diff
@@ -15,6 +15,7 @@ type Collation struct {
 
 type CollationHeader struct {
 	shardID           *big.Int        //the shard ID of the shard
+	parentHash        *common.Hash    //the hash of the parent collation
```

Comment:
Sorry for back and forth. Can we remove this until we actually need it? AFAIK, other client implementations aren't using a parent hash